apps/condo/domains/notification/helpers/sendIncomingVoIPCallNotification.js

```diff
@@ -0,0 +1,41 @@
+const { getLogger } = require('@open-condo/keystone/logging')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+
+const { VOIP_INCOMING_CALL_MESSAGE_TYPE } = require('@condo/domains/notification/constants/constants')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+
+const logger = getLogger('sendIncomingVoIPCallNotification')
+
+const prepareAndSendNotification = async (context, { userId, miniAppId, callId }) => {
+    const data = {
+        userId,
+        miniAppId,
+        callId,
+    }
+    const messageData = {
+        to: { user: { id: userId } },
+        type: VOIP_INCOMING_CALL_MESSAGE_TYPE,
+        meta: {
+            dv: 1,
+            title: 'Incoming VoIP call',
+            body: 'Hello there!',
+            data,
+        },
+        sender: { dv: 1, fingerprint: 'send-incoming-voip-call-notification.' },
+    }
+
+    const { isDuplicateMessage } = await sendMessage(context, messageData)
+
+    return (isDuplicateMessage) ? 0 : 1
+}
+
+const sendIncomingVoIPCallNotification = async ({ userId, miniAppId, callId }) => {
```

Comment:
Same about `miniAppId`

Reply:
This is just an example of how it could be used. Actual names of fields and fields count can be changed if needed. This code should be removed when proper endpoint is implemented.